modules/digitaluniverse/rendering/renderabledumeshes.cpp

```diff
@@ -255,6 +255,13 @@ bool RenderableDUMeshes::isReady() const {
         (!_renderingMeshesMap.empty() || (!_labelset.entries.empty()));
 }
 
+void RenderableDUMeshes::initialize() {
+    bool success = loadData();
```

Comment:
How about instead move the throwing into the `loadData` function and skip the return value?  In that case this would just turn into:
```
void RenderableDUMeshes::initialize() {
    loadData();
}
```

and be cleaner

Reply:
We are planning to remove this class, so I'm not sure if this minor change is worth it...